Subscription

Taterbro opened the pull request:
Made the subscription page
Added the plan component
Buttons don't do anything at the moment since the APIs haven't been set up yet.

Taterbro committed:
made skeleton for card

Taterbro committed:
Update Plan.tsx

Taterbro committed:
fetched menuitems

yyyyyup

Taterbro committed:
components done

got components working

Taterbro committed:
testing responsiveness

Taterbro committed:
dynamic testing: the sequel

Taterbro committed:
done perhaps

Taterbro committed:
revamp

yt responsiveness before i work on it. This is how it looks BEFORE i start implementing the changes

Taterbro committed:
finishing touches

made all the proper responsive adjustments

Taterbro committed:
defaulted main and app.tsx

removed the changes i made in these two files

Taterbro committed:
main.tsx

added the <Router/> component back

Taterbro committed:
reduced Icon Size

- made the plans a bit closer to better fit the figma design
- reduced the size of the icon on mobile to make it less low res

Kamba56 committed:
Merge branch 'dev' into subscription

Kamba56 commented on src/Pages/subscription/assets/bgImage.png:
Kindly note that all image assets should be in the `src/assets/images` folder and all styles should be in `src/assets/styles` folder

Kamba56 commented on src/Pages/subscription/assets/subscription.css:
Kindly remove empty files

Kamba56 commented on src/Pages/subscription/components/Button.tsx:
All components should be in the `src/components` folder. You can create a subscription folder in the components folder and add your components there. Only the subscription page component should be in the `src/pages` folder

Kamba56 reviewed (changes requested):
I can't view your page, kindly make use of the router created. You can find it in the `src/router/index.tsx` file. Your route(s) should be a child of the <PrivateRoute />.

Please do not create a new PR or branch after implementing changes, just push all implemented changes to this branch and we will use the same PR for the next review.

Taterbro committed:
moved

moved images in subscription to src/assets/images
moved components to src/components and created a subscription folder for subscription page components

Taterbro committed:
Merge branch 'subscription' of https://github.com/Kamba56/Migasuto2 into subscription

Kamba56 commented on src/router/index.tsx:
there seems to be a confusion about the routing, but I'll fix it. 
And I still can't render your page.

Let's forget about the privateRoute for now and just add the route separately like the history route.

Taterbro committed:
page route

made subscription page NOT protected

Kamba56 reviewed (approved):
Good job so far.
